tox.ini

```diff
@@ -31,8 +31,8 @@ commands =
     python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
     pip-sync requirements/tests.txt
     pip install -e .
-    flake8
     coverage run -p -m pytest tests
     coverage combine
     coverage xml
     coverage report
+    flake8
```

Comment:
Why do you want to move it to the end? `flake` is faster than unit tests, so running it before them allows to stop the process immediately without having to wait for tests.

Reply:
I really can not remember. You are right, will change it back.